grammar.js

```diff
@@ -1334,6 +1340,10 @@ module.exports = grammar({
         ['!=', 'binary_relation'],
         ['>=', 'binary_relation'],
         ['>', 'binary_relation'],
+        [$.keyword_like, 'pattern_matching'],
+        [seq($.keyword_not, $.keyword_like), 'pattern_matching'],
+        [seq($.keyword_similar, $.keyword_to), 'pattern_matching'],
```

Comment:
You can use your anonymous nodes here:

```suggestion
        [$._similar_to, 'pattern_matching'],
```

Reply:
Done